<ui version="4.0" >
 <class>QfrmConfigSlidePadraoClass</class>
 <widget class="QDialog" name="QfrmConfigSlidePadraoClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Slide Padr√£o</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QScrollArea" name="scrollArea" >
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>523</width>
        <height>394</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QLabel" name="lblFundo" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="btnAbrirImagem" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text" >
      <string>Abrir Imagem</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="btnSalvarImagem" >
     <property name="text" >
      <string>Salvar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="btnCancelar" >
     <property name="text" >
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>QfrmConfigSlidePadraoClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>413</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel" >
     <x>417</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
